src/test/ui/implied-bounds/compare-impl-method-projections.rs

```diff
@@ -0,0 +1,47 @@
+// Same as ./compare-impl-method.rs but we shouldn't use implied bounds from projections in impl
+// header. See `mod exploit` for why.
```

Comment:
@compiler-errors It turned out there's a good reason why `ocx.assumed_wf_types` doesn't include unnormalized projections :wink: .

Reply:
Oh, I'm aware of the current issue with `assumed_wf_types`, but I'm assuming that even more crates will fail if we don't add both unnormalized *and* normalized, but then enforce unnormalized types are WF in #105483.

Though I'm not sure if @lcnr is working on the fix for that they mentioned in the comment in the body of `ObligationCtxt::assumed_wf_types`.

Reply:
I'm assuming #105483 replaces a large unsoundness hole with a smaller unsoundness hole, so :shrug: